src/solvers/dgsem/basis_lobatto_legendre.jl

```diff
@@ -504,6 +504,13 @@ function gauss_lobatto_nodes_weights(n_nodes::Integer)
   nodes = zeros(n_nodes)
   weights = zeros(n_nodes)
 
+  # Special case for polynomial degree zero (first order finite volume)
+  if n_nodes == 1
+    nodes[1] = 0
+    weights[1] = 2
+    return nodes, weights
+  end
```

Comment:
I am not a super big fan of hijacking the `gauss_lobatto_nodes_weights()` function to return the center point, where most (all?) literature define the Gauss Lobatto nodes to always include both end points of the interval.

Would it be possible to, e.g., instead directly modify the `LobattoLegendreBasis` type to have a special case for `n_nodes == 1`? In that case one could argue that all DG node distributions will reduce to the center point for a polynomial degree of zero.

Either way, I think we need some documentation of this feature such that it is clear what it means to have a Gauss-Lobatto basis for polydeg=0.

Reply:
We could have this `if` block where it is called when constructing the basis - and possibly everywhere else, e.g., the `analyzer`. We should check it, but I think this is the minimally invasive approach.

Where/how would you suggest to document it?

Reply:
One possible place could be for the `LobattoLegendreBasis` type. Other than that, I don't have a good idea at the moment. I just feel like it would be good to have this documented, since it is strictly not a Gauss Lobatto node anymore.

Reply:
@ranocha  I added a comment there and commited it. But now there is another commit containing the previous changes you did. Can/should I withdraw it?

Reply:
No, that's fine as it is. The merge commit you created is correct. We will squash-merge the PR, so there is no need to clean up the git history manually.